app/lib/helpers/helpers.ts

```
@@ -0,0 +1,288 @@
+  });
+}
+
+/**
+ * Function to parse currency decimals
+ * @param price The price of the nft
+ * @param decimals The decimal point of the currency
+ * @returns price in decimal format
+ */
+export const parseCurrencyDecimals = (price: bigint, decimals: number) => {
+  if (decimals) {
+    return parseInt(price.toString()) / 10 ** decimals;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)